ci-operator/jobs/openshift/cluster-dns-operator/openshift-cluster-dns-operator-master-presubmits.yaml

```diff
@@ -313,6 +313,85 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-ovn-single-node,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Should we make `always_run` as false as not all changes will be related to tech-preview? We can run the test using `/test` whenever this test is needed to be run.

I had added similar test for cluster-network-operator here: https://github.com/openshift/release/pull/51201/files#diff-a1c0d571415439fe740da85b8546937cad301f1fe6adbf9c91efc365dc5f4164R1416

Reply:
In the early stages, let's have it `always_run: true` for visibility.  Automatic CI is the de facto standard.